build-calico-resource.sh

```diff
@@ -47,16 +47,13 @@ function fetch_and_validate() {
   fi
 }
 
-wget \
-  https://github.com/projectcalico/calico/releases/download/$calico_version/release-$calico_version.tgz
-tar -xf release-$calico_version.tgz
-
```

Comment:
This file is REALLY big --  over 1G... inside it are two files we use and LOTS we don't